python/GafferUI/Backups.py

```diff
@@ -62,6 +62,7 @@ def __init__( self, applicationRoot ) :
 		self.__settings["frequency"] = Gaffer.IntPlug( defaultValue = 5, minValue = 0 )
 		self.__settings["fileName"] = Gaffer.StringPlug( defaultValue = "${script:directory}/.gafferBackups/${script:name}-backup${backup:number}.gfr" )
 		self.__settings["files"] = Gaffer.IntPlug( defaultValue = 10, minValue = 1 )
+		self.__settings["enableAutoLoad"] = Gaffer.BoolPlug( defaultValue = True )
```

Comment:
I'm not sure `autoload` is the best term here - to me it implies that something might happen automatically without user interaction. Perhaps `checkForBackups`?